rktl_perception/include/rktl_perception/focus_vis_node.h

```diff
@@ -0,0 +1,50 @@
+/* Node to detect the ball's position on the field.
+ * License:
+ *   BSD 3-Clause License
+ *   Copyright (c) 2020, Autonomous Robotics Club of Purdue (Purdue ARC)
+ *   All rights reserved.
+ */
+
+#pragma once
```

Comment:
This is completely fine and works perfectly but I just don't like it and I'm leaving this comment out of pure spite.

Reply:
I'm leaving it for the same reason. idek what it does, James had it in code that I copied and pasted a year ago.